src/tasks/AutoBidAuctionTask.py

```
@@ -0,0 +1,401 @@
+                        re_match_start, re_confirm, re_bid, re_skip
+                    )
+                    count += 1
+                    self.info_set("已完成次数", count)
+                    self.log_info(f"拍卖完成 {count}/{max_count if max_count > 0 else '∞'}")
+
+                    if sell_interval > 0 and count % sell_interval == 0:
+                        self._sell_collections()
+                except TaskDisabledException:
+                    self.log_info("用户停止任务")
+                    raise
+                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)